examples/cli.ini

```diff
@@ -0,0 +1,22 @@
+# This is an example of the kind of things you can do in a configuration file
```

Comment:
I would add a comment that you can set (at least (1)) all flags that are visible through `--help all`.

(1) since manual is hidden, `--manual-*` is hidden from `--help all` even